app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java

```diff
@@ -154,6 +154,14 @@ public void onSelection(MaterialDialog dialog, View view, int which, CharSequenc
         });
     }
 
+    public ArrayList<String> getSelectedFilePath() {
```

Comment:
I won't be able to get the files selected using checkbox from the view files list outside ViewFilesAdapter. Or else i'll have to make the data memeber mDeleteName of ViewFilesAdapter public so that I can pass it as an argument outside this class.

Reply:
Leave it as is then.